src/sidebar/services/test/session-test.js

```diff
@@ -172,30 +172,26 @@ describe('SessionService', () => {
 
       it('should cache the returned profile data', () => {
         const session = createService();
-        return session
-          .load()
-          .then(() => {
-            return session.load();
-          })
-          .then(() => {
-            assert.calledOnce(fakeApi.profile.read);
-          });
+        session.load();
+        session.load();
```

Comment:
`session.load()` is an async method so the test needs to await the result. Otherwise the ongoing async activity might interrupt some other test.

Reply:
Indeed.